cmd/consensus/main.go

```diff
@@ -550,6 +561,90 @@ func main() {
 			// created with sealing engine
 			return receiptRequester, nil
 		}).
+		Component("DKG messaging engine", func(node *cmd.FlowNodeBuilder) (module.ReadyDoneAware, error) {
+
+			// brokerTunnel is used to forward messages between the DKG
+			// messaging engine and the DKG broker/controller
+			dkgBrokerTunnel = dkgmodule.NewBrokerTunnel()
+
+			// messagingEngine is a network engine that is used by nodes to
+			// exchange private DKG messages
+			messagingEngine, err := dkgeng.NewMessagingEngine(
+				node.Logger,
+				node.Network,
+				node.Me,
+				dkgBrokerTunnel,
+			)
+			if err != nil {
+				return nil, fmt.Errorf("could not initialize DKG messaging engine: %w", err)
+			}
+
+			return messagingEngine, nil
+		}).
+		Component("DKG reactor engine", func(node *cmd.FlowNodeBuilder) (module.ReadyDoneAware, error) {
+
+			// the viewsObserver is used by the reactor engine to subscribe to
+			// new views being finalized
+			viewsObserver := gadgets.NewViews()
+			node.ProtocolEvents.AddConsumer(viewsObserver)
+
+			// keyDB is used to store the private key resulting from the node's
+			// participation in the DKG run
+			keyDB := badger.NewDKGKeys(node.Metrics.Cache, node.DB)
+
+			machineAccountInfo, err := loadEpochQCPrivateData(node)
```

Comment:
```suggestion
			machineAccountInfo, err := loadMachineAccountInfo(node)
```
We should rename this, since it is not QC-specific.